routes/web.php

```diff
@@ -28,3 +28,9 @@
 Route::get('/editprofile', 'UserController@editprofile')->name('editprofile');
 
 Route::post('/saveprofile/{id?}', 'UserController@saveprofile')->name('saveprofile');
+
+Route::post('/popup','ModalController@popup')->name('popup');
+
+Route::get('/cartpage', function () {
+    return view('user.cart');
```

Comment:
```suggestion
    return view('user.cart');
```
không return trong route file